harnessed_jobs/fe55_raft_acq_sim/v0/producer_fe55_raft_acq_sim.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+""" Producer script """
+
+from simulation import fake_raft
+
+# These are specific to this test
+TESTTYPE = 'FE55'
+IMGTYPES = ['BIAS', 'FE55']
+PROCESS_NAME_IN = 'vendorIngest'
+PATTERN = '*_fe55*.fits'
+
+RAFT = fake_raft.Raft.create_from_etrav(fake_raft.RAFT_ID)
```

Comment:
Rather than using these module-level constants, I'd prefer `siteUtils.getUnitId()` instead of `fake_raft.RAFT_ID`.   Since other harnessed jobs import and use `siteUtils` directly in their producer and validator scripts, also using it here makes the code easier to understand without having to switch to this special context.